### PR TITLE
Issue 25 - Seeding Database

### LINKED ISSUE #25: Seed.js doesn't seed newly created tables

### PR DESCRIPTION
**Pull request by Abazhenov**
closes #25 , now our database should be (sparsely) populated

### CODE REVIEW

**Comment by Abazhenov** [comment]
I filled in some basic values for our currently defined tables, none of the tables are associated yet so we'll have to tackle that in a separate issue once we flesh out our join tables etc.  This basic seeding should allow us to tell basic API calls

File: seed.js [hunk 1]
```diff
@@ -17,36 +17,187 @@ name in the environment files.
...
+    return Promise.all(creatingCoupons);
+
+};
+
+
 db.sync({ force: true })
-    .then(function () {
-        return seedUsers();
-    })
+    .then(() => seedUsers())
```
Comment (zekenie) [inline seed.js]: could these run concurrently?

File: server/db/index.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ module.exports = db;
 
 // eslint-disable-next-line no-unused-vars
 var User = require('./models/user');
+var Square = require('./models/square');
```
Comment (zekenie) [inline server/db/index.js]: const

File: seed.js [hunk 3]
```diff
@@ -17,36 +17,187 @@ name in the environment files.
...
+            date: '2016-03-04',
+            rating: 0
+        },
+        {
+            title: 'I Love Japanese!!',
+            content: `Cause it's awesome!`,
+            date: '2016-03-05',
+            rating: 5
+        },
+        {
```
Comment (zekenie) [inline seed.js]: how do these get associated with products?

File: server/db/index.js (hunk 2 above)
Comment (Abazhenov) [inline server/db/index.js]: fixed

File: seed.js (hunk 1 above)
Comment (Abazhenov) [inline seed.js]: added a Promise.all

File: seed.js (hunk 3 above)
Comment (Abazhenov) [inline seed.js]: Opened #28

**Pull request merged**